src/System.Runtime/tests/System/Array.cs

```diff
@@ -1068,6 +1094,22 @@ public static void TestLastIndexOf()
     }
 
     [Fact]
+    public static void TestLastIndexOf_Invalid()
+    {
+        Assert.Throws<ArgumentNullException>(() => Array.LastIndexOf((int[])null, 1, 0, 0));
+        Assert.Throws<ArgumentOutOfRangeException>(() => Array.LastIndexOf(new int[] { }, 1, 1, 1));
+        Assert.Throws<ArgumentOutOfRangeException>(() => Array.LastIndexOf(new int[] { }, 1, -1, 1));
+        Assert.Throws<ArgumentOutOfRangeException>(() => Array.LastIndexOf(new int[] { }, 1, 0, 1));
+        Assert.Equal(-1, Array.LastIndexOf(new int[] { }, 1, -1, 0));
+        Assert.Equal(-1, Array.LastIndexOf(new int[] { }, 1, 0, 0));
```

Comment:
It's strange to have valid calls in an "Invalid" test method.  Seems like these should be in the previous test method, near https://github.com/dotnet/corefx/pull/7562/files#diff-6587b3aed5d02722b19b270fcc219893R1074.